v2/streaming-data-generator/src/test/java/com/google/cloud/teleport/v2/templates/StreamingDataGeneratorTest.java

```diff
@@ -161,12 +161,12 @@ public void testMessageGenerator_returnsFakeMessage() throws IOException {
 
   /** Tests generation of fake Json data message with attributes. */
   @Test
-  public void testJsonMessageGenerator_WithAttributes_returnsFakeMessageContaingAttributes()
```

Comment:
Instead of modifying the existing test can you add another test

Reply:
Done.